### ISSUE TITLE
Inconsistent enum member casing

### ISSUE THREAD

**Issue by rhelmot** [mabe02/lanterna#290]
Hi! I've been using lanterna for a school project, and I have many pull requests to submit in the next few days when I'm finished, but this is just to note something I've noticed--

The library is pretty inconsistent on whether its enum members are EnumClass.ALLCAPS or EnumClass.CamelCase, for example LinearLayout.Alignment.Beginning versus GridLayout.Alignment.BEGINNING. This should probably be fixed, though I don't know which one it ought to be, or how to change it without badly breaking all existing code using them.

**Comment by avl42** [comment]
You're right, it is not the same style throughout all enums.
Personally I don't find it worth the trouble to make it consistent.

There are some enums whose fields could be turned to ALLCAPS and an "alias"-names added:
```
    // list of constants:
    CLOSE_ON_EXIT_PRIVATE_MODE, ...;
    // alias for constants:
    @Deprecated
    public static final CloseOnExitPrivateMode = CLOSE_ON_EXIT_PRIVATE_MODE;
```
That would allow external code that just passed the constant to a method to not need any change at all.

These enums would be rather painless to change as long as CamelCase aliases are added:
-  com.googlecode.lanterna.terminal.swing.TerminalEmulatorAutoCloseTrigger
-  com.googlecode.lanterna.gui2.dialogs.MessageDialogButton
-  com.googlecode.lanterna.gui2.LinearLayout$Alignment
-  com.googlecode.lanterna.gui2.table.TableCellBorderStyle
-  com.googlecode.lanterna.gui2.Borders$BorderStyle

But there is this one exception KeyType that is used a lot in switches in Lanterna and likely also in user code, and the alias-trick would not work with switches, so changing item names in this enum could be expected to break lots of code.

If you think that a single exception is still better than many exceptions to the style guide, then this could be the way to go, but (imho) KeyType is taboo.

**Comment by mabe02** [comment]
Yeah, it's inconsistent, unfortunately. I'd rather keep it the way it is and rename it on a future non-backward compatible version.

**Comment by klaushauschild1984** [comment]
Are the KeyType constants in more use than for switch-cases? I agree that this is the main purpose. We can do the alias trick as int's. That should be compatible and break not a whole bunch of code.

**Comment by avl42** [comment]
As there are expected uses for switch cases in user code, then any other uses don't really matter.  Also, in Java (unlike C) you cannot simply mix enum values with int's.

**Comment by klaushauschild1984** [comment]
You are right. It was a bad idea.
